extensions/rescuetime-focus-session-trigger/package-lock.json

```
@@ -0,0 +1,8087 @@
+{
+  "name": "rescueTimeFocusSessionTrigger",
```

```suggestion
  "name": "rescuetime-focus-session-trigger",
```
Just to match package.json